tts-mode/hooks/ttsmode/silence.go

```
@@ -0,0 +1,145 @@
+		i = 10 + size
+	}
+	for ; i+4 <= len(audio); i++ {
+		if !isSync(audio[i:]) {
+			continue
+		}
+		frameLen, frameDur, ok = frameShape(audio[i : i+4])
+		if !ok {
+			continue
+		}
+		next := i + frameLen + int(audio[i+2]>>1&1)
+		if next != len(audio) && !(next < len(audio) && isSync(audio[next:])) {
```

golangci-lint (2.13.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)